moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp

```diff
@@ -426,6 +427,70 @@ void MotionPlanningFrame::sceneUpdate(planning_scene_monitor::PlanningSceneMonit
     planning_display_->addMainLoopJob(boost::bind(&MotionPlanningFrame::populateCollisionObjectsList, this));
 }
 
+void MotionPlanningFrame::addPrimitiveShape()
+{
+  static const double MIN_VAL = 1e-6;
+  static const std::map<std::string, shapes::ShapeType> SHAPES_MAP = {
+    { "box", shapes::BOX }, { "sphere", shapes::SPHERE }, { "cone", shapes::CONE }, { "cylinder", shapes::CYLINDER }
```

Comment:
Would't that be redundant since there are already is import buttons?

Reply:
I had a closer look, and you're right: the "Import File" button actually accepts an STL file. I just assumed it's a scene file or something, since nothing tells you about the expected format. I'll fix that in a separate PR.